### ISSUE TITLE
LC0044 - False Positive on FlowFields

### ISSUE THREAD

**Issue by Arthurvdv** [StefanMaron/BusinessCentral.LinterCop#471]
I'm unsure if the rule should raised in a scenario with FlowFields.

```AL
table 50100 MyTableA
{
    fields
    {
        field(1; MyField; Integer) { }
        field(2; "MyFlowField A"; Integer)
        {
            FieldClass = FlowField;
            CalcFormula = count(Customer);
        }
    }
}

table 50101 MyTableB
{
    fields
    {
        field(1; MyField; Integer) { }
        field(2; "MyFlowField B"; Integer)
        {
            FieldClass = FlowField;
            CalcFormula = count(Customer);
        }
    }
}

procedure MyProcedure()
var
    MyTableA: Record MyTableA;
    MyTableB: Record MyTableB;
begin
    MyTableA.TransferFields(MyTableB);
end;
```

Can we just skip all fields where `FieldClass` is set to `FlowFilter` or `FlowField`? 🤔

**Comment by jwikman** [comment]
I don't think that `FlowField` to `FlowField` should raise this rule. They will be skipped anyway, and does not make any sense in `TransferFields()`.

But there is also another interesting scenario, from `FlowField` to normal field.

```
table 50100 MyTableA
{
    fields
    {
        field(1; MyField; Integer) { }
        field(2; "MyField A"; Integer)
        {
        }
    }
}

table 50101 MyTableB
{
    fields
    {
        field(1; MyField; Integer) { }
        field(2; "MyFlowField B"; Integer)
        {
            FieldClass = FlowField;
            CalcFormula = count(Customer);
        }
    }
}

procedure MyProcedure()
var
    MyTableA: Record MyTableA;
    MyTableB: Record MyTableB;
begin
    MyTableB.CalcFields("MyFlowField B");
    MyTableA.TransferFields(MyTableB);
end;
```

In above scenario, "MyField A" would not get any value - even if "MyFlowField B" has a value. Should the rule be raised here?

**Comment by Arthurvdv** [comment]
FlowFields and FlowFilters are excluded in the (pre)release version of [v0.30.11](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.30.11) of the LinterCop, whichs resolved this issue.
